src/ocm/components/AddHosts/InventoryAddHost.tsx

```diff
@@ -19,28 +37,38 @@ const InventoryAddHosts: React.FC = () => {
   }
 
   return (
-    <>
-      <TextContent>
-        <DiscoveryInstructions showAllInstructions />
-        <Text component="p">
-          <DiscoveryImageModalButton
-            ButtonComponent={Button}
+    <Stack hasGutter>
+      <StackItem>
+        <TextContent>
+          <DiscoveryInstructions showAllInstructions />
+          <Text component="p">
+            <DiscoveryImageModalButton
+              ButtonComponent={Button}
+              cluster={cluster}
+              idPrefix="bare-metal-inventory-add-host"
+            />
+          </Text>
+          <InformationAndAlerts
             cluster={cluster}
-            idPrefix="bare-metal-inventory-add-host"
+            HostRequirementsContent={AddHostRequirementsContent}
+            setDiscoveryHintModalOpen={setDiscoveryHintModalOpen}
           />
-        </Text>
-        <InformationAndAlerts
+        </TextContent>
+      </StackItem>
+      <StackItem>{isArmArchitecture(cluster) && armArchAlert}</StackItem>
```

Comment:
The StackItem is redundant when `isArmArchitecture(cluster)` is false 
```suggestion
      {isArmArchitecture(cluster) && <StackItem>armArchAlert</StackItem>}
```